pages/index.tsx

```diff
@@ -6,10 +6,10 @@ const Index = () => {
   useEffect(() => {
     const { REALM } = process.env
     const mainUrl = REALM ? `/dao/${REALM}` : '/realms'
-    router.push(mainUrl)
+    router.replace(mainUrl)
```

Comment:
switched to replace so it that it doesn't add an extra page in the browser history

Reply:
:+1: